charts/ditto/requirements.yaml

```diff
@@ -11,6 +11,6 @@
 ---
 dependencies:
   - name: mongodb
-    repository: https://kubernetes-charts.storage.googleapis.com
+    repository: https://charts.bitnami.com/bitnami
     version: 7.6.3
```

Comment:
```suggestion
    version: ~7.14.7
```
Should we use the same version as Hono defines?

Reply:
Well, I am glad to align the versions. However, you will need to tell me if Ditto will work with it ...

Reply:
Yes, it will - this references the latest MongoDB 4.2 which Ditto 1.5.0 was explicitly tested against.

Reply:
done